CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Brushable Histogram
 
-## 1.1.6 (2018/12/26)
+## 1.2.0 (TBD)
```

Comment:
Do we really need to put this as a minor? We actually didn't add any new functionality (which is what defined a minor)

Reply:
You're right